cypress/integration/01-organization_sign_up_spec.js

```diff
@@ -183,9 +183,6 @@ describe("Organization sign up", () => {
         },
         {
           rules: {
```

Comment:
@georgehager correct me if I'm wrong, but doesn't this mean we can just call `cy.checkA11y()` here and get rid of the exclude/rules section entirely? (Goes for all the e2e tests affected)

Reply:
yep! it's not necessary to have anything passed to `checkA11y()` if there are no violations. So could be cleaned up by removing the empty rules.